pkg/cli/cert/cert.go

```diff
@@ -34,6 +34,20 @@ const (
 	k3sServerService         = "-server"
 )
 
+var services = []string{
```

Comment:
a more efficient way to do this would be to make this a `map[string]struct{}` or `map[string]bool` and then just see if the service name is in the map instead of having to walk the whole list each time, but this is fine.